src/Symfony/Component/Config/Definition/NodeInterface.php

```diff
@@ -59,11 +63,13 @@ public function hasDefaultValue();
     public function getDefaultValue();
 
     /**
-     * Normalizes the supplied value.
+     * Normalizes a value.
      *
-     * @param mixed $value The value to normalize
+     * @param mixed $value Value to normalize
```

Comment:
maybe `A value to normalize` ?

Reply:
in fact.. i think i should revert this one as the interface was already using different syntax:

https://github.com/symfony/symfony/pull/26335/files#diff-41913ce7a681852774440044e2c091bdR92